openlibrary/utils/__init__.py

```diff
@@ -24,7 +24,7 @@ def str_to_key(s: str) -> str:
 T = TypeVar('T')
```

Comment:
Ah this is the spot that was left; it seems like there are only 2 other methods here that use `T`. If we update those to use the new `[T]` syntax, we can drop this here as well!

Reply:
Fixed this in a commit :+1: Be sure to pull!